rest-api/spring-rest/src/main/java/uk/gov/gchq/gaffer/rest/config/FactoryConfig.java

```diff
@@ -61,7 +61,7 @@ public void setToSystemProperties() {
                             .filter(s -> s.startsWith("gaffer") && checkedProperties.add(s)) // Skips expensive property lookup
                             .forEach(s -> System.setProperty(s, environment.getProperty(s)));
                 } else {
-                    LOGGER.info("Skipping Property source " + propertySource);
+                    LOGGER.info("Skipping Property source {}", propertySource);
```

Comment:
Just wondering why this was used here and not further down the file:
https://github.com/gchq/Gaffer/blob/bc546709739ea2b004bf2019fa155eff84fab19f/rest-api/spring-rest/src/main/java/uk/gov/gchq/gaffer/rest/config/FactoryConfig.java#L97
https://github.com/gchq/Gaffer/blob/bc546709739ea2b004bf2019fa155eff84fab19f/rest-api/spring-rest/src/main/java/uk/gov/gchq/gaffer/rest/config/FactoryConfig.java#L109

Reply:
@t511203 This was part of your commit. Personally, I think it should probably be used there if possible.

Reply:
I am not too fussed, was just wondering. Seems quite inconsistent throughout the code base already, could just add a cleanup to make them all use the new syntax after this already large PR?

Reply:
I have created a ticket to track fixing this everywhere: https://github.com/gchq/Gaffer/issues/2618